packages/playwright-test/src/util.ts

```diff
@@ -278,3 +278,67 @@ export function getPackageJsonPath(folderPath: string): string {
   return result;
 }
 
+export const levenshteinDistance = (s: string, t: string) => {
+  // See https://en.wikipedia.org/wiki/Levenshtein_distance for reference.
+  // This is a DP implementation variant.
+  const n = s.length;
+  const m = t.length;
+
+  if (n === 0)
+    return m;
+  if (m === 0)
+    return n;
+
+  const d: number[][] = [];
+  // m rows, n cols
+  for (let i = 0; i < m + 1; i++) {
```

Comment:
why +1's here?

Reply:
We burn a column (with initialization values), so above, left, and diag are always valid. It maps to this picture upon initialization:  

```
      w e i r d
    0 1 2 3 4 5
    ___________
  0|0 1 2 3 4 5 
f 1|1
u 2|2
n 3|3
```